uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manager-leave-calendar.js

```diff
@@ -26,6 +26,7 @@ define([
     var parentCtrl = $controller('CalendarCtrl');
```

Comment:
I think manager_calendar is still fine because still it shows the calendar for the managees. Admin is getting all contacts because he manages all.